Update cursorai label

jschwager opened the pull request:
Removed appNewVersion as Cursor updated their changelog webpage which can no longer shows full version numbers and therefore can no longer be used.

All questions must be filled out or your Pull Request will be closed for lack of information. The first three questions should be answered `Yes` before submitting the pull request.
---
**Have you confirmed this pull request is not a duplicate?**
Yes
**Is this pull request creating or modifying a label in the fragments/labels folder, and not Installomator.sh itself?**
Yes
**Did you use [our editorconfig file](https://github.com/Installomator/Installomator/wiki/Contributing-to-Installomator)?**
Yes
**Additional context** Add any other context about the label or fix here.

**Installomator log** At the bottom of this pull request, provide a log of a label run by running Installomator in Terminal and saving the output. `DEBUG=1` can be enabled but **do not enable [Debug logging level](https://github.com/Installomator/Installomator/wiki/Configuration-and-Variables#logging-level) and please format the log [using a code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks)!**

Fixes #2570

```
./assemble.sh cursorai
2025-06-05 14:21:20 : INFO  : cursorai : Total items in argumentsArray: 0
2025-06-05 14:21:20 : INFO  : cursorai : argumentsArray: 
2025-06-05 14:21:20 : REQ   : cursorai : ################## Start Installomator v. 10.8beta, date 2025-06-05
2025-06-05 14:21:20 : INFO  : cursorai : ################## Version: 10.8beta
2025-06-05 14:21:20 : INFO  : cursorai : ################## Date: 2025-06-05
2025-06-05 14:21:20 : INFO  : cursorai : ################## cursorai
2025-06-05 14:21:20 : DEBUG : cursorai : DEBUG mode 1 enabled.
2025-06-05 14:21:21 : INFO  : cursorai : Reading arguments again: 
2025-06-05 14:21:21 : DEBUG : cursorai : name=Cursor
2025-06-05 14:21:21 : DEBUG : cursorai : appName=
2025-06-05 14:21:21 : DEBUG : cursorai : type=dmg
2025-06-05 14:21:21 : DEBUG : cursorai : archiveName=
2025-06-05 14:21:21 : DEBUG : cursorai : downloadURL=https://downloads.cursor.com/production/53b99ce608cba35127ae3a050c1738a959750865/darwin/universal/Cursor-darwin-universal.dmg
2025-06-05 14:21:21 : DEBUG : cursorai : curlOptions=
2025-06-05 14:21:21 : DEBUG : cursorai : appNewVersion=
2025-06-05 14:21:21 : DEBUG : cursorai : appCustomVersion function: Not defined
2025-06-05 14:21:21 : DEBUG : cursorai : versionKey=CFBundleShortVersionString
2025-06-05 14:21:21 : DEBUG : cursorai : packageID=
2025-06-05 14:21:21 : DEBUG : cursorai : pkgName=
2025-06-05 14:21:21 : DEBUG : cursorai : choiceChangesXML=
2025-06-05 14:21:21 : DEBUG : cursorai : expectedTeamID=VDXQ22DGB9
2025-06-05 14:21:21 : DEBUG : cursorai : blockingProcesses=Cursor
2025-06-05 14:21:21 : DEBUG : cursorai : installerTool=
2025-06-05 14:21:21 : DEBUG : cursorai : CLIInstaller=
2025-06-05 14:21:21 : DEBUG : cursorai : CLIArguments=
2025-06-05 14:21:21 : DEBUG : cursorai : updateTool=
2025-06-05 14:21:21 : DEBUG : cursorai : updateToolArguments=
2025-06-05 14:21:21 : DEBUG : cursorai : updateToolRunAsCurrentUser=
2025-06-05 14:21:21 : INFO  : cursorai : BLOCKING_PROCESS_ACTION=tell_user
2025-06-05 14:21:21 : INFO  : cursorai : NOTIFY=success
2025-06-05 14:21:21 : INFO  : cursorai : LOGGING=DEBUG
2025-06-05 14:21:21 : INFO  : cursorai : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-06-05 14:21:21 : INFO  : cursorai : Label type: dmg
2025-06-05 14:21:21 : INFO  : cursorai : archiveName: Cursor.dmg
2025-06-05 14:21:21 : DEBUG : cursorai : Changing directory to /Users/jaredschwager/Documents/Repositories/Installomator/build
2025-06-05 14:21:21 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2025-06-05 14:21:21 : WARN  : cursorai : No previous app found
2025-06-05 14:21:21 : WARN  : cursorai : could not find Cursor.app
2025-06-05 14:21:21 : INFO  : cursorai : appversion: 
2025-06-05 14:21:21 : INFO  : cursorai : Latest version not specified.
2025-06-05 14:21:21 : INFO  : cursorai : Cursor.dmg exists and DEBUG mode 1 enabled, skipping download
2025-06-05 14:21:21 : DEBUG : cursorai : DEBUG mode 1, not checking for blocking processes
2025-06-05 14:21:21 : REQ   : cursorai : Installing Cursor
2025-06-05 14:21:21 : INFO  : cursorai : Mounting /Users/jaredschwager/Documents/Repositories/Installomator/build/Cursor.dmg
2025-06-05 14:21:22 : DEBUG : cursorai : Debugging enabled, dmgmount output was:
expected   CRC32 $4E567E07
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/Cursor Installer

2025-06-05 14:21:22 : INFO  : cursorai : Mounted: /Volumes/Cursor Installer
2025-06-05 14:21:22 : INFO  : cursorai : Verifying: /Volumes/Cursor Installer/Cursor.app
2025-06-05 14:21:22 : DEBUG : cursorai : App size: 606M	/Volumes/Cursor Installer/Cursor.app
2025-06-05 14:21:26 : DEBUG : cursorai : Debugging enabled, App Verification output was:
/Volumes/Cursor Installer/Cursor.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: Hilary Stout (VDXQ22DGB9)

2025-06-05 14:21:26 : INFO  : cursorai : Team ID matching: VDXQ22DGB9 (expected: VDXQ22DGB9 )
2025-06-05 14:21:26 : INFO  : cursorai : Installing Cursor version 0.46.8 on versionKey CFBundleShortVersionString.
2025-06-05 14:21:26 : INFO  : cursorai : App has LSMinimumSystemVersion: 10.15
2025-06-05 14:21:26 : DEBUG : cursorai : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2025-06-05 14:21:26 : INFO  : cursorai : Finishing...
2025-06-05 14:21:29 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2025-06-05 14:21:29 : WARN  : cursorai : No previous app found
2025-06-05 14:21:29 : WARN  : cursorai : could not find Cursor.app
2025-06-05 14:21:29 : REQ   : cursorai : Installed Cursor, version 0.46.8
2025-06-05 14:21:29 : INFO  : cursorai : notifying
2025-06-05 14:21:29 : DEBUG : cursorai : Unmounting /Volumes/Cursor Installer
2025-06-05 14:21:29 : DEBUG : cursorai : Debugging enabled, Unmounting output was:
"disk4" ejected.
2025-06-05 14:21:29 : DEBUG : cursorai : DEBUG mode 1, not reopening anything
2025-06-05 14:21:29 : REQ   : cursorai : All done!
2025-06-05 14:21:29 : REQ   : cursorai : ################## End Installomator, exit code 0
```

jschwager committed:
Update cursorai.sh

Removed appNewVersion as Cursor updated their changelog webpage which can no longer shows full version numbers and therefore can no longer be used.

gilburns commented:
I would propose having two working labels both with an appNewVersion. They both are currently offering the same download link, but in previous weeks when it was 0.xx, they offered separate versions.

**Cursor Stable Branch:**
```
cursorai|\
cursoraistable)
    name="Cursor"
    type="dmg"
    updateFeed=$(curl -fsL  "https://www.cursor.com/api/download?platform=darwin-universal&releaseTrack=stable")
    appNewVersion=$(getJSONValue "${updateFeed}" "version")
    downloadURL=$(getJSONValue "${updateFeed}" "downloadUrl")
    expectedTeamID="VDXQ22DGB9"
    ;;

```

**Cursor Latest Branch:**
```
cursorailatest)
    name="Cursor"
    type="dmg"
    updateFeed=$(curl -fsL  "https://www.cursor.com/api/download?platform=darwin-universal&releaseTrack=latest")
    appNewVersion=$(getJSONValue "${updateFeed}" "version")
    downloadURL=$(getJSONValue "${updateFeed}" "downloadUrl")
    expectedTeamID="VDXQ22DGB9"
    ;;

```

jschwager commented:
@gilburns Good catch. I don't think I had realized that the API download link page also listed the version number. That's a much better method than scraping the changelog page since the version number at that link should always match the version of the download being offered. I get the feeling that they may switch that stable link over to the 1.0 version within short order. I'm just going to close this PR in the meantime.

jschwager committed:
Updated label to utilize JSON feed for appNewVersion

jschwager commented:
Updated assemble output:
`./assemble.sh cursorai      
2025-06-09 13:52:23 : INFO  : cursorai : Total items in argumentsArray: 0
2025-06-09 13:52:23 : INFO  : cursorai : argumentsArray: 
2025-06-09 13:52:23 : REQ   : cursorai : ################## Start Installomator v. 10.8beta, date 2025-06-09
2025-06-09 13:52:23 : INFO  : cursorai : ################## Version: 10.8beta
2025-06-09 13:52:23 : INFO  : cursorai : ################## Date: 2025-06-09
2025-06-09 13:52:23 : INFO  : cursorai : ################## cursorai
2025-06-09 13:52:23 : DEBUG : cursorai : DEBUG mode 1 enabled.
2025-06-09 13:52:24 : INFO  : cursorai : Reading arguments again: 
2025-06-09 13:52:24 : DEBUG : cursorai : name=Cursor
2025-06-09 13:52:24 : DEBUG : cursorai : appName=
2025-06-09 13:52:24 : DEBUG : cursorai : type=dmg
2025-06-09 13:52:24 : DEBUG : cursorai : archiveName=
2025-06-09 13:52:24 : DEBUG : cursorai : downloadURL=https://downloads.cursor.com/production/53b99ce608cba35127ae3a050c1738a959750865/darwin/universal/Cursor-darwin-universal.dmg
2025-06-09 13:52:24 : DEBUG : cursorai : curlOptions=
2025-06-09 13:52:24 : DEBUG : cursorai : appNewVersion=1.0.0
2025-06-09 13:52:24 : DEBUG : cursorai : appCustomVersion function: Not defined
2025-06-09 13:52:24 : DEBUG : cursorai : versionKey=CFBundleShortVersionString
2025-06-09 13:52:24 : DEBUG : cursorai : packageID=
2025-06-09 13:52:24 : DEBUG : cursorai : pkgName=
2025-06-09 13:52:24 : DEBUG : cursorai : choiceChangesXML=
2025-06-09 13:52:24 : DEBUG : cursorai : expectedTeamID=VDXQ22DGB9
2025-06-09 13:52:24 : DEBUG : cursorai : blockingProcesses=Cursor
2025-06-09 13:52:24 : DEBUG : cursorai : installerTool=
2025-06-09 13:52:24 : DEBUG : cursorai : CLIInstaller=
2025-06-09 13:52:24 : DEBUG : cursorai : CLIArguments=
2025-06-09 13:52:24 : DEBUG : cursorai : updateTool=
2025-06-09 13:52:24 : DEBUG : cursorai : updateToolArguments=
2025-06-09 13:52:24 : DEBUG : cursorai : updateToolRunAsCurrentUser=
2025-06-09 13:52:24 : INFO  : cursorai : BLOCKING_PROCESS_ACTION=tell_user
2025-06-09 13:52:24 : INFO  : cursorai : NOTIFY=success
2025-06-09 13:52:24 : INFO  : cursorai : LOGGING=DEBUG
2025-06-09 13:52:24 : INFO  : cursorai : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-06-09 13:52:24 : INFO  : cursorai : Label type: dmg
2025-06-09 13:52:24 : INFO  : cursorai : archiveName: Cursor.dmg
2025-06-09 13:52:24 : DEBUG : cursorai : Changing directory to /Users/jaredschwager/Documents/Repositories/Installomator/build
2025-06-09 13:52:24 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2025-06-09 13:52:24 : WARN  : cursorai : No previous app found
2025-06-09 13:52:24 : WARN  : cursorai : could not find Cursor.app
2025-06-09 13:52:24 : INFO  : cursorai : appversion: 
2025-06-09 13:52:24 : INFO  : cursorai : Latest version of Cursor is 1.0.0
2025-06-09 13:52:24 : INFO  : cursorai : Cursor.dmg exists and DEBUG mode 1 enabled, skipping download
2025-06-09 13:52:24 : DEBUG : cursorai : DEBUG mode 1, not checking for blocking processes
2025-06-09 13:52:24 : REQ   : cursorai : Installing Cursor
2025-06-09 13:52:24 : INFO  : cursorai : Mounting /Users/jaredschwager/Documents/Repositories/Installomator/build/Cursor.dmg
2025-06-09 13:52:24 : DEBUG : cursorai : Debugging enabled, dmgmount output was:
expected   CRC32 $F88CADB1
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/Cursor Installer

2025-06-09 13:52:24 : INFO  : cursorai : Mounted: /Volumes/Cursor Installer
2025-06-09 13:52:25 : INFO  : cursorai : Verifying: /Volumes/Cursor Installer/Cursor.app
2025-06-09 13:52:25 : DEBUG : cursorai : App size: 641M	/Volumes/Cursor Installer/Cursor.app
2025-06-09 13:52:28 : DEBUG : cursorai : Debugging enabled, App Verification output was:
/Volumes/Cursor Installer/Cursor.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: Hilary Stout (VDXQ22DGB9)

2025-06-09 13:52:28 : INFO  : cursorai : Team ID matching: VDXQ22DGB9 (expected: VDXQ22DGB9 )
2025-06-09 13:52:28 : INFO  : cursorai : Installing Cursor version 1.0.0 on versionKey CFBundleShortVersionString.
2025-06-09 13:52:28 : INFO  : cursorai : App has LSMinimumSystemVersion: 11.0
2025-06-09 13:52:28 : DEBUG : cursorai : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2025-06-09 13:52:28 : INFO  : cursorai : Finishing...
2025-06-09 13:52:32 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2025-06-09 13:52:32 : WARN  : cursorai : No previous app found
2025-06-09 13:52:32 : WARN  : cursorai : could not find Cursor.app
2025-06-09 13:52:32 : REQ   : cursorai : Installed Cursor, version 1.0.0
2025-06-09 13:52:32 : INFO  : cursorai : notifying
2025-06-09 13:52:32 : DEBUG : cursorai : Unmounting /Volumes/Cursor Installer
2025-06-09 13:52:32 : DEBUG : cursorai : Debugging enabled, Unmounting output was:
"disk4" ejected.
2025-06-09 13:52:32 : DEBUG : cursorai : DEBUG mode 1, not reopening anything
2025-06-09 13:52:32 : REQ   : cursorai : All done!
2025-06-09 13:52:32 : REQ   : cursorai : ################## End Installomator, exit code 0`

jschwager committed:
Merge remote-tracking branch 'upstream/main' into new-label-cursorai

jschwager committed:
Cursor download JSON file is no longer available, removed appNewVersion since no Mac version listed on download page, reworked downloadURL to scrape Mac Universal DMG from page source.

jschwager committed:
Update cursorai.sh

Cursor changed their download page again. Updated downloadURL to grab correct version.

jschwager commented:
Cursor changed their download page **again**. Updated commit to grab correct download link.
```
./assemble.sh cursorai
2025-10-06 11:07:54 : INFO  : cursorai : Total items in argumentsArray: 0
2025-10-06 11:07:54 : INFO  : cursorai : argumentsArray: 
2025-10-06 11:07:54 : REQ   : cursorai : ################## Start Installomator v. 10.9beta, date 2025-10-06
2025-10-06 11:07:54 : INFO  : cursorai : ################## Version: 10.9beta
2025-10-06 11:07:54 : INFO  : cursorai : ################## Date: 2025-10-06
2025-10-06 11:07:54 : INFO  : cursorai : ################## cursorai
2025-10-06 11:07:54 : DEBUG : cursorai : DEBUG mode 1 enabled.
2025-10-06 11:07:54 : INFO  : cursorai : Reading arguments again: 
2025-10-06 11:07:54 : DEBUG : cursorai : name=Cursor
2025-10-06 11:07:54 : DEBUG : cursorai : appName=
2025-10-06 11:07:54 : DEBUG : cursorai : type=dmg
2025-10-06 11:07:54 : DEBUG : cursorai : archiveName=
2025-10-06 11:07:54 : DEBUG : cursorai : downloadURL=https://api2.cursor.sh/updates/download/golden/darwin-universal/cursor/
2025-10-06 11:07:54 : DEBUG : cursorai : curlOptions=
2025-10-06 11:07:54 : DEBUG : cursorai : appNewVersion=
2025-10-06 11:07:54 : DEBUG : cursorai : appCustomVersion function: Not defined
2025-10-06 11:07:54 : DEBUG : cursorai : versionKey=CFBundleVersion
2025-10-06 11:07:54 : DEBUG : cursorai : packageID=
2025-10-06 11:07:54 : DEBUG : cursorai : pkgName=
2025-10-06 11:07:54 : DEBUG : cursorai : choiceChangesXML=
2025-10-06 11:07:55 : DEBUG : cursorai : expectedTeamID=VDXQ22DGB9
2025-10-06 11:07:55 : DEBUG : cursorai : blockingProcesses=Cursor
2025-10-06 11:07:55 : DEBUG : cursorai : installerTool=
2025-10-06 11:07:55 : DEBUG : cursorai : CLIInstaller=
2025-10-06 11:07:55 : DEBUG : cursorai : CLIArguments=
2025-10-06 11:07:55 : DEBUG : cursorai : updateTool=
2025-10-06 11:07:55 : DEBUG : cursorai : updateToolArguments=
2025-10-06 11:07:55 : DEBUG : cursorai : updateToolRunAsCurrentUser=
2025-10-06 11:07:55 : INFO  : cursorai : BLOCKING_PROCESS_ACTION=tell_user
2025-10-06 11:07:55 : INFO  : cursorai : NOTIFY=success
2025-10-06 11:07:55 : INFO  : cursorai : LOGGING=DEBUG
2025-10-06 11:07:55 : INFO  : cursorai : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-10-06 11:07:55 : INFO  : cursorai : Label type: dmg
2025-10-06 11:07:55 : INFO  : cursorai : archiveName: Cursor.dmg
2025-10-06 11:07:55 : DEBUG : cursorai : Changing directory to /Users/jaredschwager/Documents/Repositories/Installomator/build
2025-10-06 11:07:55 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2025-10-06 11:07:55 : WARN  : cursorai : No previous app found
2025-10-06 11:07:55 : WARN  : cursorai : could not find Cursor.app
2025-10-06 11:07:55 : INFO  : cursorai : appversion: 
2025-10-06 11:07:55 : INFO  : cursorai : Latest version not specified.
2025-10-06 11:07:55 : REQ   : cursorai : Downloading https://api2.cursor.sh/updates/download/golden/darwin-universal/cursor/ to Cursor.dmg
2025-10-06 11:07:55 : DEBUG : cursorai : No Dialog connection, just download
2025-10-06 11:08:01 : INFO  : cursorai : Downloaded Cursor.dmg – Type is  zlib compressed data – SHA is ada07e1813bb1d8d6d0b0674c8112cae6329d200 – Size is 246532 kB
2025-10-06 11:08:01 : DEBUG : cursorai : DEBUG mode 1, not checking for blocking processes
2025-10-06 11:08:01 : REQ   : cursorai : Installing Cursor
2025-10-06 11:08:01 : INFO  : cursorai : Mounting /Users/jaredschwager/Documents/Repositories/Installomator/build/Cursor.dmg
2025-10-06 11:08:05 : DEBUG : cursorai : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified   CRC32 $8ECFF7FA
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified   CRC32 $64B39EBF
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified   CRC32 $D45154EA
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified   CRC32 $00000000
Checksumming disk image (Apple_HFS : 4)…
disk image (Apple_HFS : 4): verified   CRC32 $4BF3A650
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified   CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified   CRC32 $D45154EA
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified   CRC32 $CE737FE8
verified   CRC32 $F5B0A713
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/Cursor Installer

2025-10-06 11:08:05 : INFO  : cursorai : Mounted: /Volumes/Cursor Installer
2025-10-06 11:08:05 : INFO  : cursorai : Verifying: /Volumes/Cursor Installer/Cursor.app
2025-10-06 11:08:05 : DEBUG : cursorai : App size: 666M	/Volumes/Cursor Installer/Cursor.app
2025-10-06 11:08:09 : DEBUG : cursorai : Debugging enabled, App Verification output was:
/Volumes/Cursor Installer/Cursor.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: Hilary Stout (VDXQ22DGB9)

2025-10-06 11:08:09 : INFO  : cursorai : Team ID matching: VDXQ22DGB9 (expected: VDXQ22DGB9 )
2025-10-06 11:08:09 : INFO  : cursorai : Installing Cursor version 1.7.33 on versionKey CFBundleVersion.
2025-10-06 11:08:09 : INFO  : cursorai : App has LSMinimumSystemVersion: 11.0
2025-10-06 11:08:09 : DEBUG : cursorai : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2025-10-06 11:08:09 : INFO  : cursorai : Finishing...
2025-10-06 11:08:12 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2025-10-06 11:08:12 : WARN  : cursorai : No previous app found
2025-10-06 11:08:12 : WARN  : cursorai : could not find Cursor.app
2025-10-06 11:08:12 : REQ   : cursorai : Installed Cursor, version 1.7.33
2025-10-06 11:08:12 : INFO  : cursorai : notifying
2025-10-06 11:08:13 : DEBUG : cursorai : Unmounting /Volumes/Cursor Installer
2025-10-06 11:08:24 : DEBUG : cursorai : Debugging enabled, Unmounting output was:
"disk4" ejected.
2025-10-06 11:08:24 : DEBUG : cursorai : DEBUG mode 1, not reopening anything
2025-10-06 11:08:24 : REQ   : cursorai : All done!
2025-10-06 11:08:24 : REQ   : cursorai : ################## End Installomator, exit code 0
```

jschwager committed:
Merge branch 'Installomator:main' into new-label-cursorai

jschwager committed:
Update cursorai.sh

Re-added JSON feed, downloadURL and appNewVersion are pulled from this file

jschwager committed:
Update cursorai.sh

Cursor updated their JSON feed. The stable track no longer offers the most recent version. The latest track now offers the most up to date download.

jschwager commented:
Recent commit I decided to switch this to the "latest" release track as this aligns with the version that gets downloaded on the Cursor website when you click the "Download" button.
```
./assemble.sh cursorai
2026-02-02 16:57:41 : INFO  : cursorai : Total items in argumentsArray: 0
2026-02-02 16:57:41 : INFO  : cursorai : argumentsArray: 
2026-02-02 16:57:41 : REQ   : cursorai : ################## Start Installomator v. 10.9beta, date 2026-02-02
2026-02-02 16:57:41 : INFO  : cursorai : ################## Version: 10.9beta
2026-02-02 16:57:41 : INFO  : cursorai : ################## Date: 2026-02-02
2026-02-02 16:57:41 : INFO  : cursorai : ################## cursorai
2026-02-02 16:57:41 : DEBUG : cursorai : DEBUG mode 1 enabled.
2026-02-02 16:57:42 : INFO  : cursorai : Reading arguments again: 
2026-02-02 16:57:42 : DEBUG : cursorai : name=Cursor
2026-02-02 16:57:42 : DEBUG : cursorai : appName=
2026-02-02 16:57:42 : DEBUG : cursorai : type=dmg
2026-02-02 16:57:42 : DEBUG : cursorai : archiveName=
2026-02-02 16:57:42 : DEBUG : cursorai : downloadURL=https://downloads.cursor.com/production/4f2b772756b8f609e1354b3063de282ccbe7a69b/darwin/universal/Cursor-darwin-universal.dmg
2026-02-02 16:57:42 : DEBUG : cursorai : curlOptions=
2026-02-02 16:57:42 : DEBUG : cursorai : appNewVersion=2.4.27
2026-02-02 16:57:42 : DEBUG : cursorai : appCustomVersion function: Not defined
2026-02-02 16:57:42 : DEBUG : cursorai : versionKey=CFBundleVersion
2026-02-02 16:57:42 : DEBUG : cursorai : packageID=
2026-02-02 16:57:42 : DEBUG : cursorai : pkgName=
2026-02-02 16:57:42 : DEBUG : cursorai : choiceChangesXML=
2026-02-02 16:57:42 : DEBUG : cursorai : expectedTeamID=VDXQ22DGB9
2026-02-02 16:57:42 : DEBUG : cursorai : blockingProcesses=Cursor
2026-02-02 16:57:42 : DEBUG : cursorai : installerTool=
2026-02-02 16:57:42 : DEBUG : cursorai : CLIInstaller=
2026-02-02 16:57:42 : DEBUG : cursorai : CLIArguments=
2026-02-02 16:57:42 : DEBUG : cursorai : updateTool=
2026-02-02 16:57:42 : DEBUG : cursorai : updateToolArguments=
2026-02-02 16:57:42 : DEBUG : cursorai : updateToolRunAsCurrentUser=
2026-02-02 16:57:42 : INFO  : cursorai : BLOCKING_PROCESS_ACTION=tell_user
2026-02-02 16:57:42 : INFO  : cursorai : NOTIFY=success
2026-02-02 16:57:42 : INFO  : cursorai : LOGGING=DEBUG
2026-02-02 16:57:42 : INFO  : cursorai : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2026-02-02 16:57:42 : INFO  : cursorai : Label type: dmg
2026-02-02 16:57:42 : INFO  : cursorai : archiveName: Cursor.dmg
2026-02-02 16:57:43 : DEBUG : cursorai : Changing directory to /Users/jaredschwager/Documents/Repositories/Installomator/build
2026-02-02 16:57:43 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2026-02-02 16:57:43 : WARN  : cursorai : No previous app found
2026-02-02 16:57:43 : WARN  : cursorai : could not find Cursor.app
2026-02-02 16:57:43 : INFO  : cursorai : appversion: 
2026-02-02 16:57:43 : INFO  : cursorai : Latest version of Cursor is 2.4.27
2026-02-02 16:57:43 : INFO  : cursorai : Cursor.dmg exists and DEBUG mode 1 enabled, skipping download
2026-02-02 16:57:43 : DEBUG : cursorai : DEBUG mode 1, not checking for blocking processes
2026-02-02 16:57:43 : REQ   : cursorai : Installing Cursor
2026-02-02 16:57:43 : INFO  : cursorai : Mounting /Users/jaredschwager/Documents/Repositories/Installomator/build/Cursor.dmg
2026-02-02 16:57:44 : DEBUG : cursorai : Debugging enabled, dmgmount output was:
expected   CRC32 $60700662
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/Cursor Installer

2026-02-02 16:57:44 : INFO  : cursorai : Mounted: /Volumes/Cursor Installer
2026-02-02 16:57:44 : INFO  : cursorai : Verifying: /Volumes/Cursor Installer/Cursor.app
2026-02-02 16:57:44 : DEBUG : cursorai : App size: 875M	/Volumes/Cursor Installer/Cursor.app
2026-02-02 16:57:48 : DEBUG : cursorai : Debugging enabled, App Verification output was:
/Volumes/Cursor Installer/Cursor.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: Hilary Stout (VDXQ22DGB9)

2026-02-02 16:57:48 : INFO  : cursorai : Team ID matching: VDXQ22DGB9 (expected: VDXQ22DGB9 )
2026-02-02 16:57:48 : INFO  : cursorai : Installing Cursor version 2.0.77 on versionKey CFBundleVersion.
2026-02-02 16:57:48 : INFO  : cursorai : App has LSMinimumSystemVersion: 11.0
2026-02-02 16:57:48 : DEBUG : cursorai : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2026-02-02 16:57:48 : INFO  : cursorai : Finishing...
2026-02-02 16:57:51 : INFO  : cursorai : name: Cursor, appName: Cursor.app
2026-02-02 16:57:51 : WARN  : cursorai : No previous app found
2026-02-02 16:57:51 : WARN  : cursorai : could not find Cursor.app
2026-02-02 16:57:51 : REQ   : cursorai : Installed Cursor, version 2.0.77
2026-02-02 16:57:51 : INFO  : cursorai : notifying
2026-02-02 16:57:52 : DEBUG : cursorai : Unmounting /Volumes/Cursor Installer
2026-02-02 16:57:52 : DEBUG : cursorai : Debugging enabled, Unmounting output was:
"disk4" ejected.
2026-02-02 16:57:52 : DEBUG : cursorai : DEBUG mode 1, not reopening anything
2026-02-02 16:57:52 : REQ   : cursorai : All done!
2026-02-02 16:57:52 : REQ   : cursorai : ################## End Installomator, exit code 0
```

jschwager committed:
Merge branch 'Installomator:main' into new-label-cursorai

jschwager commented:
Sorry for the confusion above. I had renamed my branch in my fork and didn't realize it would close my PR.

uurazzle reviewed (changes requested):
Hi @jschwager,

Thanks for the PR. We noticed the label includes `blockingProcess`/`blockingProcesses` with only the app name.

For example...

```
blockingProcesses=( "Cursor" )
```

### Remove redundant blockingProcesses app name

For standard `/Applications` installs, Installomator can usually infer and handle the app process automatically, so explicitly setting the app name in `blockingProcesses` is often redundant.

Please remove unnecessary `blockingProcess`/`blockingProcesses` app-name entries unless there is a specific non-default process that really needs to be handled. Thanks!

Detected in label(s): cursorai

jschwager committed:
Removed unnecessary blockingProcess

uurazzle reviewed (approved):
Hi @jschwager,

Thanks for contributing this Installomator label ([#2421: Update cursorai label](https://github.com/Installomator/Installomator/pull/2421))—everything looks good. Appreciate you taking the time to add it to the project!

gilburns reviewed (approved):
Thank you for your contribution to the project!
